### DISCUSSION TITLE
Traducción errónea "Filtrado Anisotropico"

### DISCUSSION THREAD

**Discussion by caldivargas** [mupen64plus-ae/mupen64plus-ae#1004] [General]
![Screenshot_20220214-193056](https://user-images.githubusercontent.com/78030406/153975759-0fa063b7-c44b-4a45-ae2c-aa41efa35bf9.jpg)
No es un bug, en la descripción de la opción del filtrado Anisotropico tiene la descripción de Remove halos around ....
![Screenshot_20220214-193428](https://user-images.githubusercontent.com/78030406/153976077-65434956-66aa-4f1c-a9e0-1345dd05f3dc.jpg)

**Comment by fzurita** [comment]
Gracias, lo arreglaré.

**Reply by caldivargas** [reply]
Gracias por responder. Tengo otra petición. Acerca de Audio Buffer Size, creo que existe una opción en la versión de PC que asigna el Buffer de audio automático de acuerdo al dispositivo. ¿Se puede implementar también en la versión de Android?

**Comment by fzurita** [comment]
Yo ya hago una forma de eso automáticamente si seleccionas "estirar el audio"

**Reply by caldivargas** [reply]
Entiendo. Aún así escuchaba un ruido blanco. No sé si llamarlo así, como estática. Usé una aplicación para detectar los ms de buffer y resolví el problema. El que viene por defecto en  Buffer Size es de 64 ms, lo modifiqué a 75 y se escucha más nitido. Gracias por responder.
